recipes/ibis-framework/meta.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: ibis-framework
+  version: "0.7.1"
+
+source:
+  git_url: https://github.com/cloudera/ibis
+  git_rev: v0.7.1
```

Comment:
Could we please use the [PyPI release]( https://pypi.python.org/pypi/ibis-framework/0.7.1 ) instead?